pkg/api/applications/v2/subscribe.go

```diff
@@ -52,6 +52,8 @@ type PollingSubscriber struct {
 	// Adjust the poll duration by a random amount. Defaults to 1.0, effectively
 	// a random amount up to the full poll interval.
 	JitterFactor float64
+	// Flag indicating that failed activities should still be reported.
+	ReportFailedActivities bool // TODO Should this be part of the ActivityFeedQuery?
```

Comment:
I think that todo sounds better, reduces unnecessary data being sent.

Reply:
@parsenz where should we track these kinds of changes?